crates/starknet/src/starknet/add_invoke_transaction.rs

```diff
@@ -33,21 +35,55 @@ pub fn add_invoke_transaction(
     Ok(transaction_hash)
 }
 
+pub fn add_invoke_transaction_v3(
+    starknet: &mut Starknet,
+    broadcasted_invoke_transaction: BroadcastedInvokeTransactionV3,
+) -> DevnetResult<TransactionHash> {
+    // TODO: check if bounds sum is equal to 0
```

Comment:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/e0186d915ae2a1c959208771530d11b6c7b89a8e.
Removed the comment, because the function `is_max_fee_zero_value` handles the check. The bounds sum is the total value of L1 fee + L2 fee. Its purpose is to check if there is value send for fee, not if its enough